README.md

```diff
@@ -34,215 +42,199 @@ parameters:
+kubectl apply -f jujucluster.yaml
 ```
 
-After creating the sample resources, you can check the controller-manager logs again and see that it is doing work such as bootstrapping a Juju controller and creating other resource dependencies. 
-Eventually it should create a model, at which point reconciliation should stop.
-```sh
- manager 1.673380499749945e+09    INFO    Created model    {"response": {"ModelInfo":{"Name":"jujucluster-sample","UUID":"3e086dd2-2014-42a7-8685-1ba5dce6abc2","Type":"iaas","ControllerUUID":"ba7d6d4b-a892-4fb1-8500-cc904a48b75b", │
-│ manager 1.6733804997502618e+09    INFO    Stopping reconciliation    {"controller": "jujucluster", "controllerGroup": "infrastructure.cluster.x-k8s.io", "controllerKind": "JujuCluster", "JujuCluster": {"name":"jujucluster-sample"
-```
+The Charmed Kubernetes cluster will deployed. It may take some time for the machines to come up and for the charms to be fully deployed. You can check their current status using the following command:
 
-### Connecting the deployed controller to a local Juju client
-The cluster controller creates a secret containing a Juju registration string that you can use to connect your local Juju client. Assuming your cluster is in the default namespace, you can list all secrets:
 ```sh
-kubectl get secrets
+kubectl get machines
 ```
 
-The registration secret is named `<your-cluster-name>-registration-secret`. List the contents of the secret.
+You can also check the jujucluster status, which will contain the status of the Juju model:
+
 ```sh
-kubectl get secret <your-cluster-name>-registration-secret -o jsonpath='{.data}'
+kubectl describe jujucluster jujucluster
 ```
 
-The output of the command above will look similar to the following:
+Once all machines are in the running phase, provisioning will be complete. Note that at this point your control plane will not be ready, as a CNI will need to be deployed. 
+
+### Connecting the cluster to a local Juju client
+
+Some tasks (such as deployment of CNI charms or general debugging) may require a local Juju client be connected to the Juju controller that is deployed with the cluster. Doing so will require you run run the `juju register` command, and pass it a registration string. 
+
+The cluster controller creates a secret containing a Juju registration string that you can use to connect your local Juju client. Assuming your cluster is in the default namespace, you can list all secrets:
+
 ```sh
-{"registration-string":"<encoded_data>"}
+kubectl get secrets
 ```
 
+The registration secret is named `<your-cluster-name>-registration-string`. 
 The encoded registration data is contained in the key `registration-string`. You can decode the data contained in that key with the following command:
+
 ```sh
-echo '<encoded_data>' | base64 --decode
+kubectl get secret jujucluster-registration-string -o jsonpath='{.data.registration-string}' | base64 --decode
```

Comment:
I think @addyess had a pattern I liked in some docs, using an environment variable in steps like these, so later they can be applied in later commands, It gets around the messy <things likr this> and makes it easier to disambiguate similar sounding things. So 
```
THING=`command stuff`
```
blah blah well written blah blah
```
nextcommand $THING
```

sort of construction